packages/osd-stylelint-plugin-stylelint/src/utils/extract_regex.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const extractRegex = (value: string) => {
```

Comment:
this seems slightly brittle - there's not an off-the-shelf tool we can use here?

Reply:
This has worked for everything I've tested so far, and I wanted to write some actual unit tests for this, but it seems setting up testing will be a little bit more involved for this package. I'll take a look to see if there are any other tools we can use.

Reply:
@BSFishy I've added a follow up task here: https://github.com/opensearch-project/oui/issues/848